product_brand/views/product_brand_view.xml

```diff
@@ -147,7 +147,7 @@
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_kanban_view" />
         <field name="arch" type="xml">
-            <xpath expr="//strong[hasclass('o_kanban_record_title')]" position="after">
+            <xpath expr="//div[hasclass('d-flex')]" position="after">
```

Comment:
Avoid to do generic search on elements with class `d-flex`, like you are doing here:
`expr="//div[hasclass('d-flex')]"`

Try to use something more direct, like for example:
 `expr="//field[@name='name']/.."` (putting `/..` after the field will have the same effect as the expression above)

Reply:
Thanks, Done